kitipy/context.py

```diff
@@ -92,87 +96,7 @@ def with_stack(self, stack_name: str):
-
-        Args:
-            cmd (str): The command to run.
-            **kwargs: See Executor.run() options for more details.
-
-        Raises:
-            paramiko.SSHException:
-                When the SSH client fail to run the command. Note that this
-                won't be raised when the command could not be found or it
-                exits with code > 0 though, but only when something fails at
-                the SSH client/server lower level.
-        
-        Returns:
-            subprocess.CompletedProcess
-        """
-        return self.executor.run(cmd, **kwargs)
-
-    def local(self, cmd: str, **kwargs) -> subprocess.CompletedProcess:
-        """Run a command on local host.
-        
-        This method is particularly useful when you want to run some commands
-        on local host whereas the Executor is running in remote mode. For
-        instance, you might want to check if a given git tag or some Docker 
-        images exists on a remote repository/registry before deploying it, 
-        or you might want to fetch the local git author name to log deployment
-        events somewhere. Such checks are generally better run locally.
-
-        Args:
-            cmd (str): The command to run.
-            **kwargs: See Executor.run() options for more details.
-
-        Raises:
-            paramiko.SSHException:
-                When the SSH client fail to run the command. Note that this
-                won't be raised when the command could not be found or it
-                exits with code > 0 though, but only when something fails at
-                the SSH client/server lower level.
-        
-        Returns:
-            subprocess.CompletedProcess
-        """
-        return self.executor.local(cmd, **kwargs)
-
-    def cd(self, path: str):
-        self.executor.cd(path)
-
-    def path_exists(self, path: str) -> bool:
-        return self.executor.path_exists(path)
-
-    def copy(self, src: str, dest: str):
-        """Copy a local file to a given path. If the underlying executor has
-        been configured to work in remote mode, the given source path will
-        be copied over network."""
-        self.executor.copy(src, dest)
-
-    def get_stage_names(self):
-        """Get the name of all stages in the configuration"""
-        return self.config['stages'].keys()
-
-    def get_stack_names(self):
-        """Get the name of all stacks in the configuration"""
-        return self.config['stacks'].keys()
-
-    @property
-    def is_local(self):
-        """Check if current kitipy Executor is in local mode"""
-        return self.executor.is_local
-
-    @property
-    def is_remote(self):
-        """Check if current kitipy Executor is in remote mode"""
-        return self.executor.is_remote
-
-    @property
-    def meta(self):
-        """Meta properties from current click.Context"""
-        return click.get_current_context().meta
-
+    # @TODO: fix that
```

Comment:
possible to let a more descriptive message ?

Reply:
Haha, yeah I'd like... but I've no idea what I meant :joy: